test/e2e/csv_e2e_test.go

```diff
@@ -502,6 +502,13 @@ var _ = Describe("ClusterServiceVersion", func() {
 		})
 
 		It("remains in phase Succeeded when only one pod is available", func() {
+			Eventually(func() bool {
+				dep, err := c.GetDeployment(testNamespace, "deployment")
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(dep).ShouldNot(BeNil())
```

Comment:
nit: can we handle this error and nil check directly (e.g. `if err != nil ...`) without using gomega assertions?

Reply:
I'll change it.  Thanks.